x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/engines_overview.tsx

```diff
@@ -63,11 +67,23 @@ export const EnginesOverview: React.FC = () => {
       <EuiPageContent panelPaddingSize="s" className="enginesOverview">
         <FlashMessages />
         <EuiPageContentHeader>
```

Comment:
```suggestion
        <EuiPageContentHeader responsive={false}>
```

Super quick mobile suggestion - I _think_ the Engines title + button are small enough here that we don't need to send the button down to a new row on smaller screens